test.js

```diff
@@ -151,6 +151,12 @@ test('throws on multiple non-existing files', async t => {
 	});
 });
 
+test('does not throw when not matching any file on glob pattern', async t => {
+	fs.mkdirSync(t.context.tmp);
+
+	await t.notThrowsAsync(cpy(['*.js'], t.context.tmp));
```

Comment:
Can you also add a test that tests mixed path and glob. I don't think you currently handle that correctly.

Reply:
Never throw on `glob` patterns (even if mixed with `path`).

**or**

Throw on mixed `path` and `glob` when `path` does not exist?

Reply:
> Throw on mixed path and glob when path does not exist?

This